app/views/users/registrations/_edit_account.html.erb

```diff
@@ -19,6 +19,10 @@
   <%= f.input :name, label: t('full_name') %>
   <%= f.input :email, required: true %>
 
+  <%- if devise_mapping.confirmable? %>
```

Comment:
The confirmation email is needed to sign up. There's no way a user would see this page yet still need to confirm.